### DISCUSSION TITLE
Who hosts the togithub.com domain?

### DISCUSSION THREAD

**Discussion by bcoe** [renovatebot/renovate#18010] [Request Help]
I notice renovate using the togithub.com domain (_to prevent spam on originating PRs/issues_).

Is this domain hosted by renovate? If not is there any documentation available on who provides this service?

My concern being, what if a bad actor takes over the domain.

**Comment by rarkins** [comment]
It's hosted by the Renovate team at Mend, and overseen by me directly.

It's deployed to Netlify and literally just a one-line `_redirects` file:

```
/* https://github.com/:splat   302!
```

I guess I might "open source" this repo in case that helps anyone sleep easier at night. I would be very happy if GitHub came up with their own non-backlinking service.

**Reply by bcoe** [reply]
@rarkins we were considering using the `togithub.com` for one of my team's own automations, to solve the same problem (spam on originating PRs).

Do you run this service with the intention that others might leverage it as well?

**Reply by rarkins** [reply]
I have no objection to that, as it is not billed on traffic and likely doesn't generate much at all either.
